packages/postgres/test/test-ssl.js

```diff
@@ -0,0 +1,63 @@
+const test = require('ava');
+const keyvTestSuite = require('@keyv/test-suite').default;
+const {keyvOfficialTests, keyvIteratorTests} = require('@keyv/test-suite');
+const Keyv = require('keyv');
+const KeyvPostgres = require('this');
+const {endPool} = require('../src/pool.js');
+
+test.serial('throws if ssl is not used', async t => {
+	endPool();
+	try {
+		const keyv = new KeyvPostgres({uri: 'postgresql://postgres:postgres@localhost:5433/keyv_test'});
+		await keyv.get('foo');
+		t.fail();
+	} catch {
+		t.pass();
+	} finally {
+		endPool();
+	}
+});
+
+const options = {ssl: {rejectUnauthorized: false}};
```

Comment:
@wgwz  - is this the only option for the pool on ssl? Are there other options that need to happen for it to work? The reason I am asking is I want to build documentation on the README for others on how to setup Keyv using postgres and SSL option.